azure-pipelines.yml

```diff
@@ -15,6 +15,17 @@ pr:
 variables:
 - template: ${{ variables.Pipeline.Workspace }}/.azure-pipelines/templates/variables.yml
 
+parameters:
+- name: architectures
+  type: object
+  default:
+  - name: AMD64
+    value: amd64
+    pool: pool-ubuntu-2004
+  - name: ARM64
+    value: arm64
+    pool: ubuntu-arm64-2004-pool
```

Comment:
The naming is inconsistent with `pool-ubuntu-2004`. Why not calling it `pool-ubuntu-2004-arm64`?

Reply:
@wangzelin007 Can we change the pool name to same pattern?